xtask/src/cppdocs.rs

```diff
@@ -99,9 +99,9 @@ pub fn generate(show_warnings: bool) -> Result<(), Box<dyn std::error::Error>> {
         renderer_software: true,
         gettext: true,
         accessibility: true,
-        system_testing: false,
-        freestanding: false,
-        experimental: false,
+        system_testing: true,
+        freestanding: true,
+        experimental: true,
```

Comment:
Maybe not experimental... 
Right now that's only the test framework which is already documented anyway.
But if we add stuff as experimental, I'm not sure we want to have it in the docs. Or what do you think?

Reply:
Ah right, because https://github.com/slint-ui/slint/pull/5398 hasn't been merged.